app/views/flats/show.html.erb

```diff
@@ -26,6 +26,8 @@
 <h2>Reviews</h2>
 <ul>
   <% @flat.reviews.each do |review| %>
-    <li><%= review.content %> || <%= review.rating %></li>
+      <li>Details: <%= review.content %><br>Rating: <%= review.rating %>/10 stars</li>
   <% end %>
+
 </ul>
+<%= render "components/date_picker" %>
```

Comment:
This booking form is already added, it needs only one input. but good idea on putting it on a separate file.